data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+
+**Executing code in a Notebook:**
+Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
+Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
+Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
+If the cell type is code, and the cell contains python code, the code will be compiled and executed.
+If the cell type is markdown, then it will be rendered upon execution to yield nicely formatted text.
+(For some handy markdown tips, see `this blog post <https://medium.com/analytics-vidhya/the-ultimate-markdown-guide-for-jupyter-notebook-d5e5abf728fd>`_ or the `relevant section from the JupyterLab documentation <https://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Working%20With%20Markdown%20Cells.html#Markdown-Cells>`_). 
+
+.. figure:: /_static/notebook.png
+    :name: notebook_aspect
+
+    An example of what users see in the Notebook Aspect. The left panel is a file tree from which you can access your personal files, and which will come pre-populated with a handful of tutorial notebooks that users can execute on their own. The largest panel is the Jupyter notebook, where markdown and code can be entered and executed. You can also see that one can have multiple notebooks open in separate tabs.
+
+**What is a kernel?**
+In the RSP Notebook Aspect, your notebooks will be operating in a kernel that has access to the full Rubin Science Pipelines, including the "Butler" (see below) that will be your main access tool to extract images and catalogs from the DP0.1 data.
+Many standard Python libraries and modules will be available, and users can install additional Python tools they wish to use.
+In DP0.1, the Notebook Aspect will not offer access to queries from the :ref:`Tools-RSP-Portal`. 
+
+**Is all the code in Python?**
+Yes, the RSP Notebook Aspect will only have python environments for DP0.
+To access DP0.1 data from the Notebook Aspect, users will need to use Python commands and code.
+Much of the LSST Science Pipelines code is in Python, and the DP0 :ref:`Examples-DP0-1-Notebooks` use Python as well.
+These tutorials contain executable examples of the commands required to access and analyze data.
+All DP0 delegates should feel free to copy and paste from the provided tutorials.
+Anyone new to Python and looking to learn more might benefit from this `Python for Beginners <https://www.python.org/about/gettingstarted>`_ website (which includes links to tutorial in a variety of languages).
+Web searches for "python *(thing you want to do)*" are usually pretty successful too. 
+
+**What is the Butler?**
+The Butler is a middleware component of the Data Management System (DMS) for persisting and retrieving image datasets.
```

Comment:
Of course the Butler can deal with all kinds of data, not just images.  Catalogs, PSF models, WCS, masks, etc. are all accessible via the butler